packages/components/modals/NotEnoughFundModal.tsx

```diff
@@ -28,13 +28,14 @@ export const NotEnoughFundsModal: FC<{
   cost?: Coin;
   label?: string;
   visible?: boolean;
+  address?: string;
   onClose?: () => void;
-}> = ({ label = "Not enough funds", cost, visible, onClose }) => {
+}> = ({ label = "Not enough funds", cost, visible, onClose, address }) => {
   const selectedWallet = useSelectedWallet();
   const selectedNetwork = useSelectedNetworkInfo();
   const { balances } = useBalances(
     selectedNetwork?.id,
-    selectedWallet?.address,
+    address || selectedWallet?.address,
   );
```

Comment:
we should pass an user id, not just an address, we don't know if the selectedNetwork matches this address' network

I would recommend to remove
```
const selectedWallet = useSelectedWallet();
const selectedNetwork = useSelectedNetworkInfo();
```
and instead take and `user id` or `network id + address` as props

see: https://github.com/TERITORI/teritori-dapp/blob/main/CONTRIBUTING.md#multi-networks-multi-chains

Reply:
we could also just skip checking the funds for the DAO case since the funds can change between proposal creation and execution, it should only be a warning anyway

Reply:
I propose i do this in the next PR to keep small PR
What i can do is a warning at the moment of execution, for me it does not make sense to warn or block this at the proposal creation but should be at the execution, what do you think ?

Like warning: DAO does not have enough funds, execution is likely to fail. Do you still want to execute it ?

Reply:
[53793c3](https://github.com/TERITORI/teritori-dapp/pull/1336/commits/53793c3f32338cbb5eec4614aa486875066f4419)

Reply:
> for me it does not make sense to warn or block this at the proposal creation but should be at the execution, what do you think 
yes I agree sorry if it wasn't clear